apidoc/Titanium/App/iOS/UserDefaults.yml

```diff
@@ -0,0 +1,257 @@
+---
+name: Titanium.App.iOS.UserDefaults
+summary: |
+    The UserDefaults module is used for storing application-related data in property/value pairs 
+    that persist beyond application sessions and device power cycles. UserDefaults allows the suiteName
+    of the UserDefaults to be specified at creation time.
+description: |
+    Unlike Titanium.App.Properties, Titanium.App.iOS.UserDefaults does not pull properties defined in the `tiapp.xml` file.
+
+    To create a notification action, use the <Titanium.App.iOS.createUserDefaults> method.
+creatable: false
+extends: Titanium.Proxy
+since: "4.0.0"
+
+properties:
+  - name: suiteName
+    summary: Sets the name of the suite to be used to access UserDefaults.
+    type: String
+    availability: creation
+
+methods:
+  - name: getBool
+    summary: Returns the value of a property as a boolean data type.
+    returns:
+        type: Boolean
+    parameters:
+      - name: property
+        summary: Name of property.
+        type: String
+        
+      - name: default
+        summary: Default value to return if property does not exist.
+        type: Boolean
+        optional: true
+        
+  - name: getDouble
+    summary: Returns the value of a property as a double (double-precision, floating point) data type.
+    description: |
+        This method must only be used to retrieve properties created with 
+        [setDouble](Titanium.App.Properties.setDouble).  
```

Comment:
Should the links be changed from `Titanium.App.Properties` to `Titanium.App.iOS.UserDefaults`?

Repeat for `getInt` and `setInt` methods.